include/indicators/block_progress_bar.hpp

```diff
@@ -175,11 +175,12 @@ class BlockProgressBar {
   std::pair<std::string, int> get_postfix_text() {
     std::stringstream os;
     const auto max_progress = get_value<details::ProgressBarOption::max_progress>();
+    progress_ = static_cast<float>(tick_)/max_progress;
```

Comment:
Is this the best place to update `progress_`?  The value of `progress_` is also used on line 260 before it's been updated.